modules/osdk-cli-reference-scorecard.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-scorecard_{context}"]
+= scorecard
+
+The `operator-sdk scorecard` command runs the scorecard tool. The command
```

Comment:
I think this needs another phrase or sentence to explain what the scorecard is meant to do.